pywinauto/unittests/test_Calendar.py

```diff
@@ -96,13 +96,13 @@ def test_should_throw_runtime_error_when_try_to_set_invalid_type_of_calendar(sel
 
     def test_should_get_valid_type_of_calendar(self):
         self.calendar.set_id('japan')
-        self.assertEqual(self.calendar.get_id, 'CAL_JAPAN')
+        self.assertIsNotNone(self.calendar.get_id)
```

Comment:
Do you check that method object is not `None`? Why? It's always not `None`. I guarantee. :)
You really smoke some weed.